Correct package name in Console Applications

andynava opened the pull request:
Microsoft.ApplicationInsights.DependecyCollector -> Microsoft.ApplicationInsights.DependencyCollector

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

andynava committed:
Correct package name in Console Applications

Microsoft.ApplicationInsights.DependecyCollector -> Microsoft.ApplicationInsights.DependencyCollector
